constants/constants.js

```diff
@@ -0,0 +1 @@
+export const TOTALS_STUDY = 'Totals'
```

Comment:
we can put this file in `constants/index.js` so we don't need to import it with the repeating `constants` directory and file.

```js
import { TOTALS_STUDY } from '../../constants/constants'
```

vs

```js
import { TOTALS_STUDY } from '../../constants'
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/137/commits/6c921e8f5dfccbd2c7666d80b951df5613819ba9